### ISSUE TITLE
How to applay colours to output panel?

### ISSUE THREAD

**Issue by szczemp** [jgclark/vscode-todo-highlight#98]
I have TODO Highlight v2 configured for highlight TODO and FIXME in php files. This works well in php files, but in the output panel everything is uncolored.
![Untitled](https://github.com/user-attachments/assets/eca0bef0-d5c1-4b5d-a331-64de6d961027)

My settings.json contains this config:

> "todohighlight.keywords": [
        {
            "text": "TODO",
            "color": "#fff",
            "selectColor": "#000",
            "backgroundColor": "rgba(255, 170, 50, .5)",
            "overviewRulerColor": "#fa3"
        },
        {
            "text": "FIXME",
            "color": "#fff",
            "backgroundColor": "rgba(255, 100, 150, .5)",
            "overviewRulerColor": "#f69"
        },
    ],
    "todohighlight.include": [
        "**/*.js",
        "**/*.jsx",
        "**/*.ts",
        "**/*.tsx",
        "**/*.html",
        "**/*.css",
        "**/*.scss",
        "**/*.php",
        "**/*.rb",
        "**/*.txt",
        "**/*.mdown",
        "**/*.md",
        "**/*.py",
        "**/*.log"
    ],
    "todohighlight.defaultStyle": {
        "isWholeLine": true,
    },

`"todohighlight.keywords": [
        {
            "text": "TODO",
            "color": "#fff",
            "selectColor": "#000",
            "backgroundColor": "rgba(255, 170, 50, .5)",
            "overviewRulerColor": "#fa3"
        },
        {
            "text": "FIXME",
            "color": "#fff",
            "backgroundColor": "rgba(255, 100, 150, .5)",
            "overviewRulerColor": "#f69"
        },
    ],
    "todohighlight.include": [
        "**/*.js",
        "**/*.jsx",
        "**/*.ts",
        "**/*.tsx",
        "**/*.html",
        "**/*.css",
        "**/*.scss",
        "**/*.php",
        "**/*.rb",
        "**/*.txt",
        "**/*.mdown",
        "**/*.md",
        "**/*.py",
        "**/*.log"
    ],
    "todohighlight.defaultStyle": {
        "isWholeLine": true,
    },`

**Comment by jgclark** [comment]
The code (which I inherited) does not apply colouring in the output panel. 
I'm turning this into a feature request, and marking as one where I would request help from others to help make it happen.

**Comment by jgclark** [comment]
The settings in your first post aren't clear, but it seems to have two different settings for PHP files, including: `"/.php",` which almost certainly won't work. Please double-check user and workspace level JSON settings.

**Comment by szczemp** [comment]
No,  I have only one config. I paste it to the post twice, because githun trims some characters.
Here is my new config fot todo highlight.
![todo](https://github.com/user-attachments/assets/502c58a6-9a76-4b7d-bb79-8108c8d6f29f)

**Comment by jgclark** [comment]
I see. Well, my earlier comment still stands. That screenshot is inherited from v1 of this plugin, and I have never seen it colour the OUTPUT panel. Please do improve the plugin if you can.
